easy_my_coop_es/models/coop.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class SubscriptionRequest(models.Model):
+    _inherit = 'subscription.request'
+
+    vat = fields.Char(
+        string='Tax ID',
+        help="""
+        The Tax Identification Number. Complete it if the contact is subjected to
+        government taxes. Used in some legal statements."
+        """
```

Comment:
Can people have a VAT in Spain? 

In Belgium, when we only need vat number (`company_register_number` in code) for _moral_ persons. Therefore, for companies, the subscription request creates a company partner (`is_company == true`) and a partner (`is_company == False`) with company as parent (cf parent_id). This partner is then the contact person for that company.

With this in mind, do you need to add that field?

Reply:
Yes, in Spain VAT is generic for company and people. The first character (a letter) is different but the VAT is present always.

Reply:
Ok then, let's merge it 👍